server/subsonic/helpers.go

```diff
@@ -317,6 +317,7 @@ func osChildFromAlbum(ctx context.Context, al model.Album) *responses.OpenSubson
 	child.DisplayAlbumArtist = al.AlbumArtist
 	child.AlbumArtists = artistRefs(al.Participants[model.RoleAlbumArtist])
 	child.ExplicitStatus = mapExplicitStatus(al.ExplicitStatus)
+	child.SortName = al.SortAlbumName
```

Comment:
This would work (and it is probably necessary anyway), but the root cause I think is that `getAlbumList2` (and at least `getStarred2`) should return `albumID3`, not `child`.

Reply:
Ah yes. I can fix the `2` variants